### ISSUE TITLE
Contributor's guide link is broken from https://chromium.googlesource.com/external/github.com/catapult-project/catapult.git

### ISSUE THREAD

**Issue by randomascii** [catapult-project/catapult#2073]
The catapult project is accessible in two places:
https://chromium.googlesource.com/external/github.com/catapult-project/catapult.git
and:
https://github.com/catapult-project/catapult

The former is more discoverable because it is what is shown by "git remote show origin" from inside Chrome's third_party\catapult directory. Unfortunately there is no clear path from this repo to the master repo on github, and the contributor's guide link is broken from the chromium.googlesource.com page. The link is:
https://chromium.googlesource.com/external/github.com/catapult-project/CONTRIBUTING.md
but it looks like it should be:
https://chromium.googlesource.com/external/github.com/catapult-project/catapult.git/+/master/CONTRIBUTING.md
or perhaps even:
https://github.com/catapult-project/catapult/blob/master/CONTRIBUTING.md

Possibly the contributor's guide link should unilaterally point to github. It also wouldn't hurt to mention that the chromium.googlesource.com repo is an upstream copy of the master version and shouldn't be directly edited.

TL;DR It looks like chromium.googlesource.com doesn't parse the (CONTRIBUTING.md) syntax in a useful way. Perhaps there needs to be a leading '/' character? I'm not sure.

**Comment by anniesullie** [comment]
@eakuefner I know you've looked a lot into .md files and how they work on chromium.googlesource.com, do you know how to do this?

**Comment by eakuefner** [comment]
@anniesullie Looks like you picked this up in https://codereview.chromium.org/1757263004; once it lands I'll merge master into md-pages so catapult.gsrc.io will be up-to-date.

**Comment by anniesullie** [comment]
Thanks, Ethan! I see your CL, is this fixed?

**Comment by anniesullie** [comment]
Silly question, I clicked through the links in the report, it is **fixed**. Closing.
